client/src/__snapshots__/Title.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Title should render the Title Component correctly 1`] = `ShallowWrapper {}`;
```

Comment:
I think the serializer might not have gone through on this one if it's only saying "ShallowWrapper"